Formula/kubescape.rb

```diff
@@ -0,0 +1,36 @@
+class Kubescape < Formula
+  desc "Kubernetes testing according to Hardening Guidance by NSA and CISA"
+  homepage "https://www.armosec.io/armo-kubescape"
+  url "https://github.com/armosec/kubescape/archive/refs/tags/v1.0.88.tar.gz"
+  sha256 "5d052f1830dcab6e660ca97dade2bcf1b00410197b767060f84f17fcaf43c7cf"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/armosec/kubescape/cmd.BuildNumber=v#{version}
+      -X github.com/armosec/kubescape/cautils/getter.ArmoBEURL=api.armo.cloud
+      -X github.com/armosec/kubescape/cautils/getter.ArmoERURL=report.euprod1.cyberarmorsoft.com
+      -X github.com/armosec/kubescape/cautils/getter.ArmoFEURL=portal.armo.cloud
+    ].join(" ")
+
+    system "go", "mod", "tidy"
```

Comment:
Do we need this? What's wrong with the `go.mod` in the tarball?

Reply:
I took that from the existing installation docs at https://github.com/armosec/kubescape#how-to-build

Let me try to build without.

Reply:
Unfortunately, it won't build without the `go mod tidy` command.

Reply:
Upstream repo does not contain `go.sum` file. It should have one otherwise the dependencies are not pinned and I don't think we should allow that. Please reach out to the authors so they pull that file into SCM and delete this line from formula

Reply:
Sure will reach out to them